lib/Root.js

```diff
@@ -2,7 +2,8 @@ import React, { PropTypes } from 'react'
 import { Redirect, Router, Route } from 'react-router'
 import { Provider } from 'redux/react'
 import { createDispatcher, createRedux, composeStores } from 'redux'
-import { loggerMiddleware, thunkMiddleware } from './middleware'
+import thunkMiddleware from 'redux/lib/middleware/thunk'
```

Comment:
The only concern I have is relying on internal library structure. I would generally be careful, especially since examples should try to show good practices.

@gaearon what do you think?